packages/create-amplify/templates/basic-auth-data/amplify/auth/resource.ts

```diff
@@ -19,6 +19,9 @@ export const auth = defineAuth({
       //   clientId: secret('LOGINWITHAMAZON_CLIENT_ID'),
       //   clientSecret: secret('LOGINWITHAMAZON_CLIENT_SECRET'),
       // }
+      // configure callback and logout URLs
+      callbackUrls: ['https://redirect.com'],
+      logoutUrls: ['https://logout.com'],
```

Comment:
Is this required in all cases now? Ideally our getting started template should create the "simplest" auth config possible. Maybe we need to make the entire `externalProviders` block a comment

Reply:
yeah we have a requirement that users provide these if they are specifying external providers

Reply:
Right, but should the basic getting started template include `externalProviders` at all?

Reply:
another interesting angle here:
how do I define different url per branch/sandbox? (does this matter?)

Reply:
I think what we decided for that would be something like `logoutUrls: process.env.AWS_BRANCH === 'prod' ? ['https://produrl.com'] : ['http://localhost:3000']` (the AWS_BRANCH env var is automatically set by amplify hosting builds)

Reply:
I've updated to comment out external providers, and use http://localhost:3000 as the default for both, see https://github.com/aws-amplify/amplify-backend/pull/856/commits/20975bcb87eadc0fe1eb59ae90b636cd03a84a4f